src/reporter-object-factory.ts

```diff
@@ -227,21 +232,71 @@ export default function reporterObjectFactory (
         async reportTestDone (name, testRunInfo): Promise<void> {
             if (rejectReport) return;
 
-            const { screenshots, videos, errs, durationMs, testId, browsers, skipped, unstable } = testRunInfo;
+            const { screenshots, videos, errs, durationMs, testId, browsers, skipped, unstable, fixture } = testRunInfo;
+            const { layoutTestingEnabled, screenshotsDir, destinationDir, comparerBaseDir }               = layoutTestingSettings;
```

Comment:
NOTE: Discussed in private several moments about naming and how we should get paths to comparer result in production version

Reply:
Fixed naming